daemon/emer-daemon.c

```diff
@@ -596,6 +699,18 @@ add_stored_events_to_builders (EmerDaemon        *self,
       return TRUE;
     }
 
+  if (has_invalid)
+    {
+      GVariant *payload = g_variant_new ("(tt)", num_variants, token);
+      g_variant_ref_sink (payload);
```

Comment:
Are you sure about that? I can see get_nullable_payload() consuming the floating ref only for the case where `has_payload == FALSE` (meaning that it sinks the ref of the "empty auxiliary payload" is it calls it), which won't be the case here as we will be passing a specific payload whose ref needs sinking at some point.